db/schema.rb

```diff
@@ -30,16 +30,16 @@
 
   create_table "devices", :force => true do |t|
     t.string   "name"
-    t.string   "description",    :limit => 100, :default => "", :null => false
+    t.string   "description",    :limit => 100,                                :default => "", :null => false
```

Comment:
I don't think we should track local changes for schema.rb...might be good to add it to gitignore and maybe have an `db/schema.example` as we have ot plots2. Thoughts?

Reply:
Completely agree